### DISCUSSION TITLE
Sending Pdf Attachment With Flutter

### DISCUSSION THREAD

**Discussion by Ulvi02** [emailjs-com/help#3] [Q&A]
I want to send a pdf document to email using EmailJS. I am directing pathfile of the pdf to the Dynamic Variable Attachment, but I am getting an unopenable pdf to my email. Thanks in advance

    body: json.encode({
      'service_id': serviceId,
      'template_id': templateId,
      'user_id': userId,
      'template_params': {
        'name': ["('${path.path}/my_invoice.pdf')"],
        'user_email': [_user?.email ?? ''],
        'name_of': '',
      }
    }

**Comment by xr0master** [comment]
Is my_invoice.pdf file located on the server or generated on the device?
If it is created on the device, you have to send it as base64 URI.
